en/editor/scene/transform-gizmo.md

```diff
@@ -57,4 +57,27 @@ Whe the rect transform tool is active：
 
 In UI elements layout, **rect transform tool** is always used to precisely control the position and length of the node's dimensions. As to the image elements that must keep original image aspect ratio, the rectangle transform tool is normally not the best choice for size adjustment.
 
+## Transform Gizmo Based Point Configure
+
+![position toggles](images/position-toggles.png)
+
+Transform Gizmo Based Point Configure is used to set the position and rotation of transform gizmo.
+
+![pivot local](images/pivot-local.png)
+![center global](images/center-global.png)
+
+### 位置
+
+Click the Pivot/Center button to toggle between Pivot and Center.
```

Comment:
Click the Pivot/Center button to toggle between Pivot and Center.
-->
Click the __Pivot/Center__ button to toggle between __Pivot__ and __Center__.